app/go.mod

```diff
@@ -19,3 +21,7 @@ require (
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 replace github.com/regen-network/regen-ledger => ../
+
+replace github.com/regen-network/regen-ledger/x/data/module => ../x/data/module
```

Comment:
just wondering if it makes sense in general to create go module in x/data/module/go.mod instead of doing that in x/data/go.mod to avoid cycle dependency issue. I guess module folder inside types, x/data, x/ecocredit, x/group is set up to mitigate this issue. Does it make sense?